src/test/statistics/chunk_statistics/histograms/histogram_utils_extended_test.cpp

```diff
@@ -0,0 +1,62 @@
+#include <string>
+
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "statistics/chunk_statistics/histograms/histogram_utils.hpp"
+
+namespace opossum {
+
+class HistogramUtilsExtendedTest : public BaseTest {
```

Comment:
These are here because they take a long time, I assume? Short doc for that, maybe?